README.md

```diff
@@ -102,8 +102,9 @@ You can deploy this project using: GitHub Pages
 
```

Comment:
**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍